jwala-integration-test/src/test/resources/selenium/test.properties

```diff
@@ -32,4 +32,5 @@ httpd.resource.deploy.path=d:/ctp/app/data/httpd
 webapp.resource.deploy.path=d:/ctp/app/webapps
 webapp.context.resource.deploy.path=${jvm.tomcatMedia.remoteDir}/${jvm.jvmName}/${jvm.tomcatMedia.rootDir}/conf/stp/localhost
 jvm.setenv.resource.deploy.path=${jvm.tomcatMedia.remoteDir}/${jvm.jvmName}/${jvm.tomcatMedia.rootDir}/bin
-jvm.server.xml.resource.deploy.path=${jvm.tomcatMedia.remoteDir}/${jvm.jvmName}/${jvm.tomcatMedia.rootDir}/conf
\ No newline at end of file
+jvm.server.xml.resource.deploy.path=${jvm.tomcatMedia.remoteDir}/${jvm.jvmName}/${jvm.tomcatMedia.rootDir}/conf
+jwala.client.name=Cerner Corporation B007 DEV
```

Comment:
If you are testing only the name, why have the other field values?

Reply:
As per other code review comments we are only using one field which shows client name. But, our solution requirement needs HHRR and datamode So, i am passing them from chef template. Now, i need to test all three values for which i can only pass from test.properties.

Reply:
Sorry, I missed this caught my eye when I was looking at the chef code. Add 3 properties then concatenate them in Jwala java code